bin/setup

```diff
@@ -136,8 +136,18 @@ docker compose up -d db
 ## build app
 docker compose --profile $profile build
 
-## setup and seed database
-docker compose --profile $profile run --rm $container bin/rails db:migrate db:seed
+## setup database
+docker compose --profile $profile run --rm $container bin/rails db:migrate
+
+## Seed database with demo data
+if [[ "$profile" = "dev" ]]; then
+  docker compose --profile dev run --rm $container bin/rails db:seed
+fi
```

Comment:
Do we not want to seed the database in offline because we're assuming it's being used for a field deployment? I saw that the user gets dropped into a setup wizard on loading the site for the first time.

Reply:
Yes, that's correct. I think I thought to leave it seeding until we were ready to actually use the onboarding flow, but enough folks have tested it now that I think it's worth being the default.